spec/support/libraryStubData.js

```diff
@@ -47,6 +53,19 @@ var libraryStubData = {
         arrayToString: function (arr) {
             return arr[0] + 'x' + arr[1];
         },
+        stringToArray: function (arr) {
+                var result = []; 
+                var temp = [];
+                var temp2 = [];
+                temp = arr.split(",");
+                for (var i = 0; i < temp.length; i++) {
+                temp2.push(temp[i].replace("x",","));
+                }
```

Comment:
why changing the `x` to a `,`, pushing onto a new array, then splitting on the `,`?  why not just do
`result.push([Number(temp[i].split("x")[0]), Number(temp[i].split("x")[1])]);` right here and skip the second array and loop?